### PR TITLE
Add support for Flutter for MacOS

### PR DESCRIPTION
**Pull request by blagoev**
This adds Flutter for MacOS support.
updates flutter example and tests apps
Fixes library loading for Flutter on MacOS
*Does not run metrics on build

### CODE REVIEW

File: .github/workflows/realm-dart-windows.yml [hunk 1]
```diff
@@ -26,10 +26,7 @@ jobs:
           submodules: 'recursive'
 
       - name: Build Ralm Dart for Windows
```
Comment (nielsenko) [inline .github/workflows/realm-dart-windows.yml:28]: Ralm should be Realm

File: .github/workflows/realm-flutter-windows.yml [hunk 2]
```diff
@@ -26,10 +26,7 @@ jobs:
           submodules: 'recursive'
 
       - name: Build Ralm Dart for Windows
```
Comment (nielsenko) [inline .github/workflows/realm-flutter-windows.yml:28]: Ralm should be Realm

Comment (blagoev) [reply .github/workflows/realm-flutter-windows.yml:28]: can you elaborate? This step name is  `Realm Dart for Windows` cause this is what it is doing. Or you refer to something else

**Review by nielsenko — CHANGES_REQUESTED** [review]
Remove the flutter_XX.png files

File: .github/workflows/realm-flutter-windows.yml (hunk 2 above)
Comment (blagoev) [reply .github/workflows/realm-flutter-windows.yml:28]: oh yeah there is a typo. got it. will fix

Comment (nielsenko) [reply .github/workflows/realm-flutter-windows.yml:28]: Minor type 'Ralm'

**Comment by blagoev** [comment]
@nielsenko These are added by the flutter tooling.

File: .github/workflows/realm-flutter-windows.yml (hunk 2 above)
Comment (nielsenko) [reply .github/workflows/realm-flutter-windows.yml:28]: Nothing to do with this PR, just noticed it

**Comment by blagoev** [comment]
@nielsenko I got your meaning. will fix

**Comment by nielsenko** [comment]
The example should work regardless. Feels a bit odd with 0 bytes files, and error screens in the repo.

**Review by nielsenko — APPROVED** [review]

File: .github/workflows/realm-flutter-windows.yml (hunk 2 above)
Comment (blagoev) [reply .github/workflows/realm-flutter-windows.yml]: fixed

**Pull request merged**